packages/components/src/internal/test/testHelpers.tsx

```diff
@@ -31,13 +31,19 @@ export interface AppContextTestProviderProps<A = AppContext> extends PropsWithCh
 export const AppContextTestProvider: FC<AppContextTestProviderProps> = props => {
     const { appContext, children, serverContext = {}, notificationContext, printLabelsContext } = props;
     const initialAppContext = useMemo(() => ({ api: getTestAPIWrapper(), ...appContext }), [appContext]);
+    const initialPrintLabelsContext = useMemo(
+        () => ({ canPrintLabels: false, ...printLabelsContext }),
+        [printLabelsContext]
+    );
 
     return (
         <ServerContextProvider initialContext={serverContext as ServerContext}>
             <AppContextProvider initialContext={initialAppContext}>
                 <GlobalStateContextProvider>
                     <NotificationsContextProvider initialContext={notificationContext as NotificationsContextState}>
-                        <LabelPrintingContextProvider initialContext={printLabelsContext as LabelPrintingContextProps}>
+                        <LabelPrintingContextProvider
+                            initialContext={initialPrintLabelsContext as LabelPrintingContextProps}
```

Comment:
nit: I think we could remove the `as LabelPrintingContextProps` here if we move it up to the useMemo part. Not a huge deal but I think doing type coercion in the JSX is messy.